bindings/go/ecosystem/ecosystem.go

```diff
@@ -0,0 +1,66 @@
+package ecosystem
```

Comment:
Let's exclude this for now, since it's not clear this should belong in the schema repository.